one/webclient.py

```diff
@@ -615,6 +615,7 @@ def _generic_request(self, reqfunction, rest_query, data=None, files=None):
                 message = json.loads(r.text)
                 message.pop('status_code', None)  # Get status code from response object instead
                 message = message.get('detail') or message  # Get details if available
+                _logger.error(message)
```

Comment:
What was the reason for adding this error log? I personally find it a bit of a nuisance to have duplicate logs. Typically all error messages are directed to stdout so this results in the same message twice

Reply:
This relates to this: https://github.com/cortex-lab/alyx/pull/829/commits/85e158b45c809c0fa5a83c464647f9d681495a92

On a 504 if Django is in debug mode that prints the django error log. This will hardly impact users but be practical for developers.

We can also downgrade the log level to debug

Reply:
Fair enough! I think a debug log might be more appropriate then